gallery/package.json

```diff
@@ -73,6 +73,7 @@
     "core-js": "^3.8.3",
     "es6-object-assign": "^1.1.0",
     "es6-set": "^0.1.5",
+    "json-loader": "^0.5.7",
```

Comment:
This doesn't seem right.  The gallery was already doing a json import before this PR

Reply:
> This doesn't seem right. The gallery was already doing a json import before this PR

Ops, I thought I had removed this.
Removed: 212a504adf72913462c9aac2c11bf991dc949cc3